aspnetcore/data/entity-framework-6.md

```diff
@@ -4,14 +4,14 @@ author: rick-anderson
 description: Entity Framework 6.3 and later works with ASP.NET Core 3.1 and later.
 ms.author: riande
 ms.custom: mvc
-ms.date: 7/24/2020
+ms.date: 7/14/2020
 no-loc: [Blazor, "Identity", "Let's Encrypt", Razor, SignalR]
 uid: data/entity-framework-6
 ---
 # ASP.NET Core and Entity Framework 6
 
-[Entity Framework](/ef/ef6/) 6.3 and later works with ASP.NET Core 3.1 and later. For more information, see the blog post [Announcing Entity Framework 6.3 with .NET Core Support](https://devblogs.microsoft.com/dotnet/announcing-entity-framework-6-3-preview-with-net-core-support/).
+[Entity Framework](/ef/ef6) 6.3 and later works with ASP.NET Core 3.1 and later. For more information, see the blog post [Announcing Entity Framework 6.3 with .NET Core Support](https://devblogs.microsoft.com/dotnet/announcing-entity-framework-6-3-preview-with-net-core-support).
```

Comment:
I've seen `@scottaddie` comment on a couple of PRs re trailing slashes so I cleaned these up too.

Reply:
Trailing slashes are ok if the server/site has a 301/302 redirect to it (i.e., avoid a network round-trip). In this case, that's exactly the situation, so I recommend reverting it here.

Reply:
@guardrex Thanks for the explanation.